features/step_definitions/purchase_steps.rb

```diff
@@ -93,11 +91,12 @@
 end
 
 Then(/^my credit card is saved for future purchases$/) do
-  expect(@user.card_token).to_not be_nil
+  sleep(5)
```

Comment:
You shouldn't need to do this. If you need some help with race conditions, let me know. I've run into them in cucumber before, but `sleep` can almost always be avoided.